packages/core/config.js

```diff
@@ -68,10 +70,13 @@ module.exports.schema = {
     message: 'should be a number ≤40',
     validate: value => intRange(0, 40)(value)
   },
-  autoBreadcrumbs: {
-    defaultValue: () => true,
-    message: 'should be true|false',
-    validate: (value) => typeof value === 'boolean'
+  enabledBreadcrumbTypes: {
+    defaultValue: () => BREADCRUMB_TYPES,
+    message: `should be null or a list of available breadcrumb types (${BREADCRUMB_TYPES.join(',')})`,
+    validate: value => value === null || (isArray(value) && reduce(BREADCRUMB_TYPES, (accum, maybeType) => {
+      if (accum === false) return accum
+      return includes(BREADCRUMB_TYPES, maybeType)
+    }, true))
```

Comment:
I think this validation logic is wrong. Consider the following failing test case:

```
  describe('schema', () => {
    ...

    describe('enabledBreadcrumbTypes', () => {
      describe('validation', () => {
        it('fails when a supplied value is not a valid breadcrumb type', () => {
	  const enabledBreadcrumbTypesValidator = config.schema.enabledBreadcrumbTypes.validate;
          expect(enabledBreadcrumbTypesValidator(['UNKNOWN_BREADCRUMB_TYPE'])).toBe(false)
        })
      })
    })
  })
```

Gives `Expected true to be false.`

Reply:
Thanks for the failing test case 🙌 

I added it to the codebase and fixed the bug (I was iterating over `BREADCRUMB_TYPES` rather than `values`).